trace/opencensus/trace/tracer/context_tracer.py

```diff
@@ -76,14 +74,19 @@ def start_span(self, name='span'):
         :rtype: :class:`~opencensus.trace.span.Span`
         :returns: The Span object.
         """
-        parent_span_id = self.span_context.span_id
+        parent_span = self.current_span()
+
+        if parent_span is None:
```

Comment:
Can you add a comment describing the django case you explained to me?

Reply:
Done.